src/Examine.Lucene/Suggest/ISuggestionExecutionContext.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Examine.Suggest;
+
+namespace Examine.Lucene.Suggest
+{
+    public interface ISuggestionExecutionContext
```

Comment:
This is public and should therefore have XML documentation - Use `/// <inheritdoc />` where appropriate

Reply:
Done